.github/workflows/expo-publish.yml

```diff
@@ -26,9 +26,8 @@ jobs:
           expo-version: 5.x
           token: ${{ secrets.expo-token }}
       - run: yarn
-      - run: cp .env.sample .env
       - run: expo publish --release-channel=${{ inputs.channel }} --non-interactive
       - uses: expo/expo-github-action/preview-comment@v7
```

Comment:
we're losing the ability to comment on the PR with the QR code of the preview app. let's consider this and if not possible atm let's create an issue to keep track of it

Reply:
issue created ✅